catalog/to-consul/resource_test.go

```diff
@@ -1261,7 +1356,7 @@ func lbService(name string) *apiv1.Service {
 			LoadBalancer: apiv1.LoadBalancerStatus{
 				Ingress: []apiv1.LoadBalancerIngress{
 					{
-						IP: "1.2.3.4",
```

Comment:
It is not necessary, that's true. I do think though that it makes the test more readable because when you assert on IP a few lines below you don't have to drill down to the function to see what IP it's setting there.